### ISSUE TITLE
picosnitch dash returns: touch ~/.config/picosnitch/dash: permission denied

### ISSUE THREAD

**Issue by Probotect0r** [elesiuta/picosnitch#16]
Just updated the package on Arch Linux (which was a mission in itself due to the python-flask-seasurf package not installing), and saw that there was a new dash command. Tried running it and got this error:

```
touch ~/.config/picosnitch/dash: permission denied
```

Took a look at the ~/.config/picosnitch folder and it was owned by root:root. I have never interacted with this folder before. I changed ownership of folder to my user. Running the command again, it says 'serving web gui...', and opens the webpage but localhost:5100 doesn't load anything. The process in the terminal also exits immediately (not sure if that's expected).

**Comment by Probotect0r** [comment]
Running `picosnitch start-dash` asks for sudo password, then returns this:

```
[sudo] password for sagar:
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/picosnitch.py", line 2168, in <module>
    sys.exit(start_picosnitch())
  File "/usr/lib/python3.10/site-packages/picosnitch.py", line 1850, in start_picosnitch
    return ui_dash()
  File "/usr/lib/python3.10/site-packages/picosnitch.py", line 1483, in ui_dash
    import dash_core_components as dcc
ModuleNotFoundError: No module named 'dash_core_components'

```

**Comment by elesiuta** [comment]
Thanks! I just released 0.11.5 which should fix this, let me know if it works.

**Comment by Probotect0r** [comment]
Running `picosnitch dash` works now. Thanks!
